app/helpers/administrate/application_helper.rb

```diff
@@ -19,6 +19,16 @@ def display_resource_name(resource_name)
         )
     end
 
+    def sort_order(order)
+      if order == "asc"
+        "ascending"
+      elsif order == "desc"
+        "descending"
+      else
+        "none"
+      end
```

Comment:
I tend not to because that `end` used for it all. Whitespacing example:

```ruby
case foo
when "bar"
  "baz"
end

case foo
  when "bar"
    "baz"
end
```

Some people don't like the four character whitespace from `"baz"` to the `end`. I think Hound won't like it either. I'm just using an extra ruby keyword to inline them; `then`.